plugins/mangatx/index.js

```diff
@@ -0,0 +1,92 @@
+var headers = {
+	'Cookie': 'nw=1',
```

Comment:
I believe you copied this from the E-Hentai plugin. `nw` means "never warn me again on those potentially offensive galleries". I guess it is not necessary for mangatx?

Reply:
yes, I forgot to clean that part. this changes included on commit 73ce376